### PR TITLE
Apply the new design for List-detail sample

### PR DESCRIPTION
**Pull request by joyl1216**
Screenshot_1606344192
Screenshot_1606345426
Screenshot_1606345397
Screenshot_1606345405
Screenshot_1606345418

### CODE REVIEW

File: ListDetail/build.gradle [hunk 1]
```diff
@@ -28,8 +28,10 @@ dependencies {
     implementation androidxDependencies.constraintLayout
     implementation androidxDependencies.ktxCore
     implementation androidxDependencies.ktxFragment
+    implementation androidxDependencies.recyclerView
 
-    implementation microsoftDependencies.dualScreenLayout
+//    implementation microsoftDependencies.dualScreenLayout
+    implementation "com.microsoft.device.dualscreen:layouts:1.0.0-alpha02"
```
Comment (joyl1216) [inline ListDetail/build.gradle:34]: Update the dependency here, since the rest of samples haven't been updated.

**Comment by joyl1216** [comment]
Add back button for the detail page in the single screen
Screenshot_1606350244
Screenshot_1606350254

**Review by Mehul1995 — APPROVED** [review]

File: ListDetail/src/main/java/com/microsoft/device/display/samples/listdetail/ItemDetailFragment.kt [hunk 2]
```diff
@@ -44,16 +32,38 @@ class ItemDetailFragment : Fragment() {
...
     }
+
+    override fun onActivityCreated(savedInstanceState: Bundle?) {
+        super.onActivityCreated(savedInstanceState)
+
+        val viewModel = ViewModelProvider(requireActivity()).get(SelectionViewModel::class.java)
+        viewModel.getSelectedItemLiveData().observe(
+            viewLifecycleOwner,
+            Observer {
+                if (it != null) {
```
Comment (CesarValiente) [inline ListDetail/src/main/java/com/microsoft/device/display/samples/listdetail/ItemDetailFragment.kt:53]: Instead this you can do it in a more kotlin way:
```
it?.let { imageView.setImageResource(it) }
```

File: ListDetail/build.gradle (hunk 1 above)
Comment (CesarValiente) [reply ListDetail/build.gradle:34]: Please Joy can you add a `//TODO message` here? so we don't miss it when we change the artifacts.

**Review by CesarValiente — APPROVED** [review]
LGTM!! Great work Joy! :-)
I've just written a couple of minor comments.

**Comment by CesarValiente** [comment]
Hey Joy I just did manual build run (it doesn't trigger automatically until we push a new commit in this branch) and the build fails:

https://github.com/microsoft/surface-duo-sdk-samples-kotlin/runs/1486031758?check_suite_focus=true

image


Please, have a look. Thanks!

**Pull request merged**